exec/cnex/cnex.c

```diff
@@ -47,15 +48,16 @@ void exec_freeExecutor(TExecutor *e)
         cell_clearCell(&e->globals[i]);
     }
     free(e->globals);
+    assert(isEmpty(e->stack));
     destroyStack(e->stack);
     free(e->stack);
     framestack_destroyFrameStack(e->framestack);
     free(e->framestack);
     free(e->module);
 
-    bytecode_freeBytecode(e->object);
+    //bytecode_freeBytecode(e->object);
```

Comment:
Why is this commented out? If this function doesn't free the bytecode, then who does?

Reply:
It's up to the caller to free the Bytecode chunk individually.  It seemed to make more sense to do it that way when modules got involved.